gui/launcher.h

```diff
@@ -124,11 +155,77 @@ class LauncherDialog : public Dialog {
 	 *
 	 * @target	name of target to select
 	 */
-	void selectTarget(const String &target);
+	virtual void selectTarget(const String &target) = 0;
+	virtual int getSelected() = 0;
 private:
 	bool checkModifier(int modifier);
 };
 
+// Disable the grid for platforms that disable fancy themes
+class LauncherChooser {
+protected:
+	LauncherDialog *_impl;
+
+public:
+	LauncherChooser();
+	~LauncherChooser();
+
+	int runModal();
+	void selectLauncher();
+};
+
+class LauncherSimple : public LauncherDialog {
```

Comment:
These sub-classes could be moved to the .cpp-file, as no other compile unit needs their definition (except the LauncherChooser's functions.